src/app/debug/table/table.component.html

```diff
@@ -1,113 +1,133 @@
+        Filter Error: {{ this.handleFilterErrorContext() }}
       </div>
-    </div>
-    <button type="button" id="UploadButton" class="btn btn-info my-2 mx-1 px-3"
-            title="Upload" (click)="uploadFileTable.click()">
-      <i class="fa fa-upload"></i>
-      <input data-cy-debug="upload" #uploadFileTable (change)="uploadReports($event)" type="file" id="uploadFileTable">
-    </button>
-    <app-button data-cy-debug="openSelected" title="Open Selected Reports" text="Open Selected" (click)="openSelected()"></app-button>
-    <app-button data-cy-debug="delete" title="Delete" icon="fa fa-trash" (click)="deleteSelected()"></app-button>
-
-    <form (keydown.enter)="$event.preventDefault();" (keyup.enter)="changeTableLimit($event)">
-      <div class="input-group">
-        <input data-cy-debug="displayAmount" type="number" class="form-control input-field"
-                 [value]="tableSettings.displayAmount" (blur)="changeTableLimit($event)">
-        <div data-cy-debug="amountShown" class="input-group-append">
-          <div class="input-group-text">/{{ metadataCount }}</div>
+    }
+    <div class="row d-flex align-items-center">
+      <app-button data-cy-debug="refresh" icon="fa fa-refresh" title="Refresh" (click)="refresh()"></app-button>
+      <app-button data-cy-debug="openSettings" icon="fa fa-cog" title="Settings" (click)="openSettingsModal()"></app-button>
+      <app-button data-cy-debug="filter" icon="fa fa-filter" title="Filter" (click)="toggleFilter()"></app-button>
+      <div ngbDropdown title="Download">
+        <button data-cy-debug="download" class="btn btn-info my-2 mx-1" id="dropdownDownloadTable" ngbDropdownToggle>
+          <i class="fa fa-download px-2"></i><span class="caret"></span>
+        </button>
+        <div ngbDropdownMenu aria-labelledby="dropdownDownloadTable">
+          <button ngbDropdownItem (click)="downloadReports(true, true)">XML & Binary</button>
+          <button ngbDropdownItem (click)="downloadReports(false, true)">XML <i class="fa fa-info-circle" title="Only a binary file can be uploaded again"></i></button>
+          <button ngbDropdownItem (click)="downloadReports(true, false)">Binary</button>
         </div>
       </div>
-    </form>
-
-    <div class="my-2 mx-1 px-2">
-      <select data-cy-change-view-dropdown class="form-control" (change)="changeView($event)">
-        <option [style.width]="viewDropdownBoxWidth" *ngFor="let view of viewSettings.views | keyvalue" [selected]="view.key === viewSettings.currentView.name" [value]="view.key">{{ view.key }}</option>
-      </select>
-    </div>
+      <button type="button" id="UploadButton" class="btn btn-info my-2 mx-1 px-3"
+              title="Upload" (click)="uploadFileTable.click()">
+        <i class="fa fa-upload"></i>
+        <input data-cy-debug="upload" #uploadFileTable (change)="uploadReports($event)" type="file" id="uploadFileTable">
+      </button>
```

Comment:
This button is not inside a form, so the type="button" can be removed, also the id is not used anywhere.
Maybe also beste to move the data-cy tag to the button, because that's also what the user will interact with.
```suggestion
      <button data-cy-debug="upload" class="btn btn-info my-2 mx-1 px-3" title="Upload (click)="uploadFileTable.click()">
        <i class="fa fa-upload"></i>
        <input #uploadFileTable (change)="uploadReports($event)" type="file" id="uploadFileTable">
      </button>
```

Reply:
added the change and adjusted the cypress e2e test for the change